### DISCUSSION TITLE
Duplicate entries in CHIME/FRB Catalog

### DISCUSSION THREAD

**Discussion by loostrum** [chime-frb-open-data/community#33] [Q&A] [answered]
The .csv version of catalog 1 from https://www.chime-frb.ca/catalog contains several entries with the same `tns_name`. These entries are not complete duplicates of each other, some of the fitted parameters differ.

After removing the duplicates, the remaining number of bursts is 535 so that seems to be correct. These are the duplicates I've found:

```WARNING: There are 2 entries for FRB20180814B
WARNING: There are 3 entries for FRB20180917A
WARNING: There are 2 entries for FRB20181019A
WARNING: There are 5 entries for FRB20181028A
WARNING: There are 2 entries for FRB20181104C
WARNING: There are 2 entries for FRB20181117B
WARNING: There are 3 entries for FRB20181119D
WARNING: There are 3 entries for FRB20181125A
WARNING: There are 2 entries for FRB20181128A
WARNING: There are 2 entries for FRB20181128C
WARNING: There are 4 entries for FRB20181222A
WARNING: There are 2 entries for FRB20181222E
WARNING: There are 2 entries for FRB20181223A
WARNING: There are 2 entries for FRB20181224E
WARNING: There are 2 entries for FRB20181226A
WARNING: There are 3 entries for FRB20181226B
WARNING: There are 2 entries for FRB20190104A
WARNING: There are 2 entries for FRB20190109A
WARNING: There are 3 entries for FRB20190111A
WARNING: There are 6 entries for FRB20190122C
WARNING: There are 2 entries for FRB20190124C
WARNING: There are 2 entries for FRB20190131D
WARNING: There are 4 entries for FRB20190208A
WARNING: There are 2 entries for FRB20190213B
WARNING: There are 2 entries for FRB20190301A
WARNING: There are 2 entries for FRB20190308B
WARNING: There are 2 entries for FRB20190308C
WARNING: There are 2 entries for FRB20190411C
WARNING: There are 3 entries for FRB20190422A
WARNING: There are 2 entries for FRB20190423A
WARNING: There are 2 entries for FRB20190423B
WARNING: There are 2 entries for FRB20190501B
WARNING: There are 2 entries for FRB20190421A
WARNING: There are 2 entries for FRB20190519A
WARNING: There are 4 entries for FRB20190519B
WARNING: There are 2 entries for FRB20190527A
WARNING: There are 2 entries for FRB20190601C
WARNING: There are 3 entries for FRB20190604F
WARNING: There are 3 entries for FRB20190605B
WARNING: There are 2 entries for FRB20190609A
WARNING: There are 2 entries for FRB20190611A
WARNING: There are 3 entries for FRB20190625E
```

**Comment by zpleunis** [comment, marked as answer]
Hi Leon,

These FRBs all have multiple components with the different rows presenting the fit parameters to the individual subbursts (TOA, intrinsic width and spectrum are provided per subburst, other parameters are shared). The `sub_num` column tracks the subbursts.

-- Ziggy

**Comment by loostrum** [comment]
Hi Ziggy, 

Thanks, that is useful to know! I was a bit confused because the .json version of the catalog is different and has just one entry per FRB.

**Comment by 0xCoto** [comment]
Also stumbled upon this and identified the sub-burst indication. Although I agree it can be a bit misleading to see these reported as 'separate' or 'unique' events depending on the observed profile, it would probably be helpful to have direct access to sub-burst properties for certain types of analysis and population studies.

(I can also confirm I had gotten **535** unique events when I cleared out the duplicates as well, so you haven't missed anything in case you were curious.)
